pallets/ethereum-transactions/src/tests/tests_eth_transaction_type.rs

```diff
@@ -2,16 +2,21 @@
 
 use crate::ethereum_transaction::*;
 use ethabi::{Function, Param, ParamType, Token};
+use sp_core::H512;
 
 pub const ROOT_HASH: [u8; 32] = [3; 32];
 pub const T2_PUBLIC_KEY: [u8; 32] = [4; 32];
+pub const T1_PUBLIC_KEY: [u8; 64] = [5u8; 64];
```

Comment:
we should be consistent over whether we specify the type or not here

Reply:
I think I get it now, you mean `[5u8; 64]` vs `[5; 64]`
I prefer the first way, it's more explicit.